spec/services/replication_transfer_updater_spec.rb

```diff
@@ -0,0 +1,266 @@
+        let(:changes) { { fixity_value: fixity_value} }
+        it_behaves_like "fixity_value_update_basics"
+        it "sets store_requested->true" do
+          update.update
+          expect(record.store_requested).to be true
+        end
+        it "does not cancel the record" do
+          update.update
+          expect(record.cancelled).to be false
+        end
+      end
+      context "we requested, fixity incorrect" do
+        let(:record) { Fabricate(:replication_transfer, bag: bag, fixity_alg: fixity_alg,
+          fixity_value: nil, from_node: Fabricate(:local_node))}
+        let(:changes) { { fixity_value: "93847238479237490238"} }
+        it_behaves_like "fixity_value_update_basics"
+        it "cancels with fixity_reject" do
+          update.update
+          expect(record.cancelled).to be true
+          expect(record.cancel_reason).to eql("fixity_reject")
+        end
+        it "does not set store_requested" do
+          update.update
+          expect(record.store_requested).to be false
+        end
+      end
+    end
+  end
+
+
+  describe ReplicationTransferUpdater::StoreRequestedUpdate do
+    let(:update) { described_class.new(record, updated) }
+    let(:record) { Fabricate(:replication_transfer, store_requested: false) }
+    let(:changes) { {store_requested: true} }
+    describe "::matching_update?" do
+      it "matches old.store_requested == false, new.store_requested == true" do
+        expect(described_class.matching_update?(record, updated)).to be true
+      end
+    end
+    describe "#update" do
+      shared_examples "store_requested_update" do
+        it "updates the record" do
+          update.update
+          expect(record.store_requested).to be true
+        end
+        it "returns true on success" do
+          allow(record).to receive(:update).and_return true
+          expect(update.update).to be true
+        end
+        it "returns false on failure" do
+          allow(record).to receive(:update).and_return false
+          expect(update.update).to be false
+        end
+        it "does not cancel the record" do
+          update.update
+          expect(record.cancelled).to be false
+        end
+        it "does not set stored" do
+          update.update
+          expect(record.stored).to be false
+        end
+      end
+      context "we are not replicating" do
+        let(:record) { Fabricate(:replication_transfer, store_requested: false, to_node: Fabricate(:node)) }
+        it_behaves_like "store_requested_update"
+        it "does not request preservation" do
+          bag_man_request = double(:bag_man_request)
+          allow(record).to receive(:bag_man_request).and_return(bag_man_request)
+          expect(bag_man_request).to_not receive(:okay_to_preserve!)
+          update.update
+        end
+      end
+      context "we are replicating" do
+        let(:record) { Fabricate(:replication_transfer, store_requested: false, to_node: Fabricate(:local_node)) }
```

Comment:
^^ Here I can clearly see the `to_node == local_node` idea in this fixture and it corresponds with the label for the context.  That is, a replication node is a local_node == to_node.  (Did I miss this stuff when I looked at fixtures above?  Or did github already update your changes while I'm reviewing all this stuff?)